packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-table.jsx

```diff
@@ -0,0 +1,101 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { CSSTransition } from 'react-transition-group';
+import { Icon, Text, ThemedScrollbars } from '@deriv/components';
+import Fieldset from 'App/Components/Form/fieldset.jsx';
+import BarriersList from './barriers-list.jsx';
+
+const BarriersTable = ({
+    barrier_1,
+    onChange,
+    setHoveredBarrier,
+    is_barriers_table_expanded,
+    turbos_barrier_choices,
+    toggleBarriersTable,
+}) => {
+    const [chosen_barrier, setChosenBarrier] = React.useState(barrier_1);
+
+    const changeBarrier = e => {
+        if (e.target.id) {
+            e.stopPropagation();
+            setHoveredBarrier(null);
+            setChosenBarrier(e.target.id);
+            onChange({
+                target: {
+                    name: 'barrier_1',
+                    value: e.target.id,
+                },
+            });
+        }
+    };
+
+    const onMouseOver = e => {
+        e.stopPropagation();
+        if (e.target.id && chosen_barrier !== e.target.id) {
+            setHoveredBarrier(e.target.id);
+        }
+    };
+
+    const onMouseOut = e => {
+        e.stopPropagation();
+        setHoveredBarrier(null);
+    };
+
+    return (
+        <CSSTransition
+            appear
+            in={is_barriers_table_expanded}
+            timeout={250}
+            classNames={{
+                appear: 'trade-container__barriers-table--enter',
+                enter: 'trade-container__barriers-table--enter',
+                enterDone: 'trade-container__barriers-table--enter-done',
+                exit: 'trade-container__barriers-table--exit',
+            }}
+            unmountOnExit
+        >
+            <Fieldset className='trade-container__fieldset trade-container__barriers-table'>
+                <div className='trade-container__barriers-table__header'>
+                    <Text color='prominent' weight='bold' size='xs'>
+                        {localize('Barriers')}
+                    </Text>
+                    <div className='trade-container__barriers-table__icon-close' onClick={toggleBarriersTable}>
+                        <Icon icon='IcCross' />
+                    </div>
+                </div>
+                <div className='trade-container__barriers-table__text'>Distance to spot</div>
+                <ThemedScrollbars>
```

Comment:
As we don't know the exact design for scroll, I don't pass property (temporary).